### PR TITLE
Fix runtime deprecated warning on Fixnum type

### PR DESCRIPTION
**Pull request by badlamer**

### CODE REVIEW

File: lib/gsl/oper.rb [hunk 1]
```diff
@@ -40,4 +40,4 @@ def /(other)
 
 end
 
-[Fixnum, Float].each { |klass| klass.send(:include, GSL::Oper) }
+[RUBY_VERSION > "2.4" ? Integer : Fixnum, Float].each { |klass| klass.send(:include, GSL::Oper) }
```
Comment (mrageh) [inline lib/gsl/oper.rb]: Perhaps you want to change this line to `RUBY_VERSION >= "2.4" `

**Comment by mrageh** [comment]
@v0dro Firstly thanks for all the work you've done on this gem, any chance we can merge this PR and release a new patch of this gem?

File: lib/gsl/oper.rb (hunk 1 above)
Comment (badlamer) [inline lib/gsl/oper.rb]: Constant RUBY_VERSION string has patch version number, example "2.4.0" and always been greater than "2.4"

Comment (mrageh) [inline lib/gsl/oper.rb]: :shipit:

Comment (david-macmahon) [inline lib/gsl/oper.rb]: Why not just use `[Integer, Float]` for all versions?

Comment (v0dro) [inline lib/gsl/oper.rb]: What if theres some outdated codebase that depends on Fixnum behaviour? IMO let this be for now we can release later versions with only Integer and Float.

**Comment by v0dro** [comment]
https://rubygems.org/gems/gsl

**Pull request merged**